nixos/modules/services/system/nscd.nix

```diff
@@ -89,14 +104,14 @@ in
 
         environment = { LD_LIBRARY_PATH = nssModulesPath; };
 
-        restartTriggers = [
+        restartTriggers = lib.optionals cfg.enableNsncd ([
```

Comment:
should this condition be `!cfg.enableNsncd`?

Reply:
Yes, you're right. Updated, thanks!